application/core/Memory_Model.php

```diff
@@ -202,7 +202,7 @@ function trailing($count = 10) {
     function some($what, $which) {
 	$results = array();
 	foreach ($this->_data as $key => $record)
-	    if ($record[$what] == $which)
+	    if ($record->$what == $which)
```

Comment:
why did you have to change the core model?

Reply:
For some reason it kept giving me a null value of $record[$what], but worked when I used the arrow.